browser/react/components/Login/Login.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+export default (props) => (
+  <div style={props.styles.container}>
+    <div>
+      <Link to="signup" style={{ 'textDecoration': 'none' }}>
```

Comment:
Can we pull this out into the styles.js file and refer to it via `props` like the other ones?